cli/command/image/build_buildkit.go

```diff
@@ -117,6 +117,22 @@ func runBuildBuildKit(dockerCli command.Cli, options buildOptions) error {
 		defer os.RemoveAll(dockerfileDir)
 	}
 
+	outputs, err := parseOutputs(options.outputs)
+	if err != nil {
+		return errors.Wrapf(err, "failed to parse outputs")
+	}
+
+	for _, out := range outputs {
+		if out.Type == "local" {
+			outDir, ok := out.Attrs["dest"]
+			if !ok {
+				return errors.Errorf("dest is required for local output")
+			}
+			delete(out.Attrs, "dest")
```

Comment:
why?

Reply:
cause dest is client side only and should not be sent to the daemon

Reply:
Perhaps add a comment to mention that